adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -157,8 +169,9 @@ - (BOOL)hasDelegateForPlacementID:(NSString *)placementID
   return result;
 }
 
-- (BOOL)canRequestBannerAdForPlacementID:(NSString *)placmentID {
-  return self.mrecPlacementID == nil || [self.mrecPlacementID isEqualToString:placmentID];
+- (BOOL)canRequestBannerAdForPlacementID:(NSString *)placmentID withBannerType:(VungleNetworkAdapterAdType)bannerType{
+    NSString *placement = (bannerType == MREC ? _mrecPlacementID : _bannerPlacementID);
```

Comment:
`self.mrecPlacementID` and `self. bannerPlacementID`

Reply:
Also, you might need to set `nil` for these 2 var after finishing playing banner/mrec ad, so adapter can load next banner/mrec with different placement ID.

Reply:
@ashinagawa Hi Aki, got it, I will use self here, and set banner/mrec placement to nil after finishing playing banner/mrec ad. Thanks for suggestions. :)